crm_claim/migrations/11.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2018 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    activity_type = env.ref('mail.mail_activity_data_todo')
+    crm_claim_model = env.ref('crm_claim.model_crm_claim')
+    cr.execute("""
+        INSERT INTO
+         mail_activity
+           (res_model, res_model_id, res_id, res_name, user_id,
+            date_deadline, summary, activity_type_id)
+        SELECT
+          %s,
+          %s,
+          id,
+          name,
+          user_id,
+          COALESCE(date_action_next, now()),
+          action_next,
+          %s
+        FROM
+          crm_claim
+        WHERE
+          date_action_next IS NOT Null
```

Comment:
I'm thinking that you can remove now this part of the WHERE

Reply:
Before this, you can set a 'date_action_next' without adding an 'action_next', if you don't add this where it won't create any activity (a 'summary' is not mandatory in the database, you can create one without text)

Reply:
Is the text mandatory on activities?

Reply:
OK, thanks for the confirmation.